wallet/wallet-cli-lib/tests/cli_test_framework.rs

```diff
@@ -320,27 +297,60 @@ impl CliTestFramework {
             vi_mode: false,
         };
 
-        let console = MockConsole::new(commands);
+        let (output_tx, output_rx) = crossbeam_channel::unbounded::<String>();
+        let (input_tx, input_rx) = crossbeam_channel::unbounded::<String>();
```

Comment:
Please be aware that crossbeam channels are not similar to boost signal in C++ (where all receivers get the message), but are more like a demultiplexer. They deliver messages only to one destination, not all. Are you sure this is what we need?

Reply:
That's fine. Anyway, we do not really need the `crossbeam-channels` here, so I replaced it with `std::sync::mpsc` (after a little refactoring).

Reply:
That's much better. Thank you. We don't need crossbeam at all. Using them can easily lead to someone calling Clone on rx, thinking that it'll act like boost signals. Using mpsc makes it much clearer. Cheers!